_includes/module-loading/bundling-webpack-cjs-npm_webpack.config.md

```diff
@@ -0,0 +1,59 @@
+1. Update the project Webpack configuration to load the {{site.productname}} CSS and to optimize the {{site.productname}} assets.
+
+    Example `webpack.config.js`
+
+    ```js
+    var path = require('path');
+    var MiniCssExtractPlugin = require('mini-css-extract-plugin');
+    var HtmlWebpackPlugin = require('html-webpack-plugin');
+
+    module.exports = {
+      mode: 'development',
+      entry: './src/index.js',
+      resolve: {
+        extensions: ['.js']
+      },
+      plugins: [
+        new MiniCssExtractPlugin(),
+        new HtmlWebpackPlugin({
+          title: 'TinyMCE Webpack Demo',
+          meta: {viewport: 'width=device-width, initial-scale=1'}
+        }),
+      ],
+      devtool: 'inline-source-map',
+      devServer: {
```

Comment:
This probably shouldn't be here as we don't mention installing webpack-dev-server.

Reply:
Removed by https://github.com/tinymce/tinymce-docs/pull/2035/commits/a47beef5c8241fc19a80b4e97ab0abb5d57abd2d